frontend/src/metabase/components/MoveQuestionsIntoDashboardsModal/MoveQuestionsIntoDashboardsInfoModal.tsx

```diff
@@ -0,0 +1,45 @@
+import { t } from "ttag";
+import _ from "underscore";
+
+import { Button, Flex, Icon, List, Modal } from "metabase/ui";
+
+interface MoveQuestionsIntoDashboardsInfoModalProps {
+  onConfirm: () => void;
+  onCancel: () => void;
+}
+
+export const MoveQuestionsIntoDashboardsInfoModal = ({
+  onConfirm,
+  onCancel,
+}: MoveQuestionsIntoDashboardsInfoModalProps) => (
+  <Modal
+    opened
+    onClose={onCancel}
+    title={t`Move saved questions into dashboards?`}
+    size="35rem"
+    withCloseButton={false}
+    data-testid="move-questions-into-dashboard-info-modal"
```

Comment:
I don't think we're using this test id anywhere. Also, usually we can access the modals by role: dialog with the name being the title.

Reply:
This is used, but `move-questions-into-dashboard-modal-close-btn` wasn't so I removed it.